src/lib/types.ts

```diff
@@ -10,6 +11,7 @@ export type Plugin = {
 	remarkPlugin?: Pluggable;
 	rehypePlugin?: Pluggable;
 	renderer?: ComponentsMap;
+	handler?: Handlers
```

Comment:
I think `remarkRehypeOptions` property is better, for customizability.
Do you think about it?

```ts
import type { Options as RemarkRehypeOptions } from 'remark-rehype';
```

```suggestion
	remarkRehypeOptions?: RemarkRehypeOptions;
```

```ts
		.use(remarkRehype, {
			allowDangerousHtml: true,
			...plugins
				.map((plugin) => plugin.remarkRehypeOptions)
				.filter(nonNullable)
				.reduce((acc, options) => ({ ...acc, ...options }), {})
		})
```

Reply:
Yes but we'd still need to do reducrion on handlers hm...

Reply:
See latest commit, I did it

Reply:
Wait sec - found issue

Reply:
It looks good! Thank you!